frontend/src/components/pages/Scheduling/DonationInformation.tsx

```diff
@@ -1,31 +1,144 @@
-import { Button, Container, HStack, Text } from "@chakra-ui/react";
+import {
+  Button,
+  Checkbox,
+  Container,
+  FormControl,
+  FormLabel,
+  HStack,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
 import React from "react";
 
+import xl from "../../../assets/donation-sizes/lg.png";
+import lg from "../../../assets/donation-sizes/md.png";
+import md from "../../../assets/donation-sizes/sm.png";
+import sm from "../../../assets/donation-sizes/xs.png";
+import customTheme from "../../../theme";
+import RadioImageSelectGroup from "../../common/RadioImageSelectGroup";
 import SchedulingProgressBar from "../../common/SchedulingProgressBar";
-import { SchedulingStepProps } from "./types";
+import { DonationSizeInterface, SchedulingStepProps } from "./types";
 
-const DonationInformation = ({
+const DonationSizes: DonationSizeInterface[] = [
+  {
+    image: sm,
+    size: "Small",
+    description: "Approximately fills one shelf of the fridge/ pantry",
+  },
+  {
+    image: md,
+    size: "Medium",
+    description: "Approximately fills two shelves of the fridge/ pantry",
+  },
+  {
+    image: lg,
+    size: "Large",
+    description:
+      "Approximately fills four shelves of the fridge/ pantry (full capacity)",
+  },
+  {
+    image: xl,
+    size: "Extra-large",
+    description:
+      "Approximately fills four shelves of the fridge/ pantry (full capacity)",
+  },
+];
+
```

Comment:
you have a duplicate here (wrong copy) - should be the following: 
![image](https://user-images.githubusercontent.com/19617248/143833843-c97b9f41-e851-4b07-955a-ca42b21086da.png)